fontforge/fvfonts.c

```diff
@@ -1522,13 +1628,13 @@ static void LayerInterpolate(Layer *to,Layer *base,Layer *other,real amount,Spli
 	LogError(_("Different stroke patterns in layer %d of %s\n"), lc, sc->name );
 
     to->splines = SplineSetsInterpolate(base->splines,other->splines,amount,sc);
-    to->refs = InterpRefs(base->refs,other->refs,amount,sc);
+    to->refs = InterpRefs(base->refs,other->refs,amount,sc->name,false);
     if ( base->images!=NULL || other->images!=NULL )
 	LogError(_("I can't even imagine how to attempt to interpolate images in layer %d of %s\n"), lc, sc->name );
 }
 
 SplineChar *SplineCharInterpolate(SplineChar *base, SplineChar *other,
-	real amount, SplineFont *newfont) {
+	real amount, SplineFont *newfont, bool only_compatible) {
```

Comment:
Does it matter that we're not doing the compatibility check in the multilayer case? I know you did some work with multilayer support before.

Reply:
(I personally have no issue with skipping multilayer support for the flag and just adding an "XXX todo" comment.)